implementations/rust/ockam/ockam_command/tests/bats/nodes.bats

```diff
@@ -41,9 +41,14 @@ force_kill_node() {
   run_success "$OCKAM" node create "$n"
 
   run_success "$OCKAM" node show "$n"
-  assert_output --partial "/dnsaddr/localhost/tcp/"
+  assert_output --partial "/dnsaddr/localhost/tcp/44041"
```

Comment:
The assigned port is random. We just want to make sure it has an address, so it's OK to revert it back.

Reply:
Reverted